s/snappy_java/build.sh

```diff
@@ -0,0 +1,46 @@
+# ----------------------------------------------------------------------------
+# Package       : lz4-java
```

Comment:
minor typo, should be: "**snappy-java**"

Reply:
You know where I got the inspiration from ;-)

So, I have updated & pushed a PR, please help review.